ServerCore/Areas/Identity/Pages/Account/Manage/_Layout.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@{ 
+    Layout = "/Areas/Identity/Pages/_Layout.cshtml";
```

Comment:
This seems to make it ignore the rest of our layout. We might still need the side navigation here, but we should still have the proper menu bar